packages/react-vapor/src/components/table-hoc/utils/TableHOCExampleUtils.tsx

```diff
@@ -0,0 +1,74 @@
+import * as moment from 'moment';
```

Comment:
I understand the point of doing this in real development. But doesn't it defeat the purpose of showing how the table works in the example page ?

Reply:
I dont want to add noise in this demo. IMO its not important to see how to add a row and other stuff when you look at the server table example but how to use the table to integrate a server side behaviour. What do you think ?

I dont like file with a lot of codes :)

Reply:
fair enough